quickwit-indexing/failpoints/mod.rs

```diff
@@ -234,25 +216,40 @@ async fn test_merge_executor_controlled_directory_kill_switch() -> anyhow::Resul
     // This tests checks that if a merger is killed in a middle of
     // a merge, then the controlled directory makes it possible to
     // abort the merging operation and return quickly.
+    // NOTE(fmassot): This test is working but not as exactly we would want.
```

Comment:
I keep this test but it's not as useful as before as now we are just doing the merge in a protected zone.